### ISSUE TITLE
typings objection, snakeCaseMappers & knexSnakeCaseMappers

### ISSUE THREAD

**Issue by cdupetit** [Vincit/objection.js#1342]
Hi,
In the definition file 'typings/objection/index.d.ts' : 
```
  const snakeCaseMappers: () => ColumnNameMappers;
  const knexSnakeCaseMappers: () => KnexMappers;

```
the options parameter is missing (see 'lib/utils/identifierMapping.js') : 
```
function knexSnakeCaseMappers(opt = {}) {}
function snakeCaseMappers(opt = {}) {}

```
The definition should be : 
```
  const snakeCaseMappers: (opt?: any) => ColumnNameMappers;
  const knexSnakeCaseMappers: (opt?: any) => KnexMappers;

```
Thanks.
Regards

**Comment by koskimas** [comment]
PR merged

**Comment by ps2goat** [comment]
When will this fix be released? It's not in 1.6.9, but it would be nice to have it.  Thanks for keeping the typing fixes coming!
